docs/resources/Audit_Log.md

```diff
@@ -29,198 +31,163 @@ Whenever an admin action is performed on the API, an entry is added to the respe
+| channel_id         | snowflake | Channel in which the entities were targeted                      | MEMBER_MOVE & MESSAGE_PIN & MESSAGE_UNPIN & MESSAGE_DELETE & STAGE_INSTANCE_CREATE & STAGE_INSTANCE_UPDATE & STAGE_INSTANCE_DELETE |
+| count              | string    | Number of entities that were targeted                            | MESSAGE_DELETE & MESSAGE_BULK_DELETE & MEMBER_DISCONNECT & MEMBER_MOVE                                                             |
+| delete_member_days | string    | Number of days after which inactive members were kicked          | MEMBER_PRUNE                                                                                                                       |
+| id                 | snowflake | ID of the overwritten entity                                     | CHANNEL_OVERWRITE_CREATE & CHANNEL_OVERWRITE_UPDATE & CHANNEL_OVERWRITE_DELETE                                                     |
+| members_removed    | string    | Number of members removed by the prune                           | MEMBER_PRUNE                                                                                                                       |
+| message_id         | snowflake | ID of the message that was targeted                              | MESSAGE_PIN & MESSAGE_UNPIN                                                                                                        |
+| role_name          | string    | Name of the role if type is `"0"` (not present if type is `"1"`) | CHANNEL_OVERWRITE_CREATE & CHANNEL_OVERWRITE_UPDATE & CHANNEL_OVERWRITE_DELETE                                                     |
+| type               | string    | Type of overwritten entity - role (`"0"`) or member (`"1"`)      | CHANNEL_OVERWRITE_CREATE & CHANNEL_OVERWRITE_UPDATE & CHANNEL_OVERWRITE_DELETE                                                     |
 
 ### Audit Log Change Object
 
+Many audit log events include a `changes` array in their [entry object](#DOCS_RESOURCES_AUDIT_LOG/audit-log-entry-object-audit-log-entry-structure). The structure for the individual changes varies based on the event type and its changed objects, so apps shouldn't depend on a single pattern of handling audit log events.
+
 ###### Audit Log Change Structure
 
+> warn
+> `APPLICATION_COMMAND_PERMISSIONS_UPDATE` and `MEMBER_ROLE_UPDATE` events don't follow the same pattern as other audit log events. Details can be found in the [change key](#DOCS_RESOURCES_AUDIT_LOG/audit-log-change-object-audit-log-change-key) section.
+
 > info
-> If `new_value` is not present in the change object, while `old_value` is, that means the property that was changed has been reset, or set to `null`
+> If `new_value` is not present in the change object while `old_value` is, it indicates that the changed property has been reset or set to `null`.
```

Comment:
```If `new_value` is not present in the change object while `old_value` is, it indicates that the changed property has been reset or set to `null`.```

Since this line now acts as the only documentation for the fact that any fields marked as `null` on their relevant change object would never be null here, would it be worth also documenting that `old_value` won't be present if the field was originally `null`?